my-videos/src/components/MyVideos/MyVideos.css

```
@@ -0,0 +1,19 @@
+.wrap {
+  display: flex;
+  flex-wrap: wrap;
+}
+.thumbnail {
+ border: 1px solid #dedede;
+}
+.thumbnail.square {
+  width: 25%;
+}
+.thumbnail.vertical {
+  width: 25%;
+}
```

we can consolidate these two styles since they're the same
```suggestion
.thumbnail.square, .thumbnail.vertical {
  width: 25%;
}
```